openlibrary/templates/subjects.html

```diff
@@ -2,7 +2,7 @@
 
 $var title: $page.name
 
-$ subject_list = [('subjects', 20), ('places', 20), ('people', 10), ('times', 10)]
+$ subject_list = [('subjects', _('SUBJECTS'), 20), ('places', _('PLACES'), 20), ('people', _('PEOPLE'), 10), ('times', _('TIMES'), 10)]
```

Comment:
Uppercasing should be done in CSS; it's generally considered bad practice to put this in the text itself, since it's not related to the content, it's just a stylistic choice.

```suggestion
$ subject_list = [('subjects', _('Subjects'), 20), ('places', _('Places'), 20), ('people', _('People'), 10), ('times', _('Times'), 10)]
```